crates/bws/src/config.rs

```diff
@@ -20,7 +20,7 @@ pub(crate) struct Profile {
     pub server_base: Option<String>,
     pub server_api: Option<String>,
     pub server_identity: Option<String>,
-    pub state_file_dir: Option<String>,
+    pub state_dir: Option<String>,
```

Comment:
Would be great to have some test coverage for this.
At the moment we just cover `server_base`

Reply:
100%, we actually already have a ticket for this: https://bitwarden.atlassian.net/browse/SM-1052

I need to go back and work through that ticket.